.github/workflows/greetings.yaml

```diff
@@ -0,0 +1,19 @@
+name: First Time Contributor Greetings
```

Comment:
A more descriptive name here will be worth it in the long run, hence the extra words.

Also renamed everything to have a `yaml` instead of the mix of `yml` and `yaml`. It doesn't matter which, but having a single extension makes grep a lot easier.